### PR TITLE
Slamdunk post added.

### PR DESCRIPTION
**Pull request by t-neumann**
@bgruening Almost feel embarrassed to still send this PR since it has been ages you asked me for it. Hope it is still fine.

### CODE REVIEW

File: _posts/2019-10-19-Slamdunk.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+site: vienna
```
Comment (bgruening) [inline _posts/2019-10-19-Slamdunk.md]: there is no site vienna, you could become an official "member site", see here: https://galaxyproject.eu/freiburg/

But until then this does not work, can you change this to "freiburg"?

File: _posts/2019-10-19-Slamdunk.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+tags: [tools]
+title: Slamdunk - SLAMseq analysis for nucleotide-conversion sequencing datasets
+---
+
+Methods to read out naturally occurring or experimentally introduced nucleic acid modifications are emerging as powerful tools to study dynamic cellular processes. The recovery, quantification and interpretation of such events in high-throughput sequencing datasets demands specialized bioinformatics approaches.
+
+One of such methods is [SLAMseq](http://doi.org/10.1038/nmeth.4435), [SLAMseq](http://doi.org/10.1038/nmeth.4435), a sequencing technology for time-resolved measurement of newly synthesized and existing RNA in cultured cells. Originally developed by the lab of [Stefan Ameres](https://www.imba.oeaw.ac.at/research/stefan-ameres/), the lab of [Johannes Zuber](https://www.imp.ac.at/groups/johannes-zuber/) extended the approach with pharmacological and chemical-genetic perturbations in order to identify direct transcriptional targets of any gene or pathway ([Muhar et al, Science 2018](http://doi.org/10.1126/science.aao2793)).
```
Comment (bgruening) [inline _posts/2019-10-19-Slamdunk.md]: SLAMseq ref is twice.

**Comment by t-neumann** [comment]
Fixed.

**Comment by bgruening** [comment]
Hi @t-neumann its online :) https://galaxyproject.eu/posts/2019/08/17/Slamdunk/

Thanks!!!

**Pull request merged**